src/system-script.js

```diff
@@ -25,13 +25,10 @@ function normalizeName(name) {
 }
 
 function getScript(address, name) {
-  const existingScripts = Array.prototype.filter.call(
-    document.querySelectorAll("script"),
-    script => script.src === address
-  );
+  const existingScript = document.querySelector(`script[src="${address}"]`)
```

Comment:
My guess is that the browser is faster than javascript is at filtering to find the correct script